guides/integrate-firebase.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Firebase
+sidebar_position: 6
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import SetupKotlin from '/docs/workflows/_sdk-setup/_setup-kotlin.mdx';
+import SetupSwift from '/docs/workflows/_sdk-setup/_setup-swift.mdx';
+import Arcade, {Clip} from '/src/components/Arcade.tsx';
+
+# Integrate Beyond Identity Passwordless Authentication into Firebase
+
+This guide provides information on how to set up Beyond Identity as a passwordless authentication provider for an application that uses [Firebase](https://firebase.google.com/docs/auth)
+
+This guide will cover:
+
+- How to configure Beyond Identity as an Identity Provider
+- How to create an identity and generate a passkey
+- How to authenticate with a passkey
+- How to configure the OpenID Connect plugin
+
+## Prerequisites
+
+- Set up a [developer account](/docs/v1/workflows/account-setup)
+
+## Firebase
+
+- [Authenticate Using OpenID Connect on Android](https://firebase.google.com/docs/auth/android/openid-connect)
+- [Authenticate Using OpenID Connect on Apple platforms](https://firebase.google.com/docs/auth/ios/openid-connect)
+
+## Install the Embedded SDK
+
+In order to use Beyond Identity functionality in your application, you will need to install the SDK. The [Android SDK](https://github.com/gobeyondidentity/bi-sdk-android) and the [Swift SDK](https://github.com/gobeyondidentity/bi-sdk-swift) provides functionality from passkey creation to passwordless authentication. A set of functions are provided to you through an `Embedded` namespace.
+
+## Initialize the Embedded SDK
+
+Once you've installed the SDK, initialize it so that you can call the Embedded functions.
+
+<Tabs groupId="sdks" queryString>
+<TabItem value="kotlin" label="Kotlin">
+
+<SetupKotlin />
+
+</TabItem>
+<TabItem value="swift" label="Swift">
+
+<SetupSwift />
+
+</TabItem>
+</Tabs>
+
+## Set up Beyond Identity as an Identity Provider
+
+To set up Beyond Identity as an Identity Provider, you need to create a [Realm](/docs/v1/workflows/realms) to hold identities and configuration. Inside that realm, you'll need to create an [Application](/docs/v1/workflows/applications) that contains the authentication flow configuration. These can be configured in you admin console that was created for you when you signed up for a developer account.
+
+### Create a Realm
+
+From the admin console, click the realm drop-down, and click **Create new realm**.
+
+<Arcade clip={Clip.CreateRealm} />
+
+### Create an Application
+
+From the admin console, click **Apps**, then click **Add app**.
+
+There is a lot to configure when creating an application. When creating your application make sure:
+
+- Redirect URIs contains your application's App Scheme or Universal URL
```

Comment:
Just checking, do we need to add some Firebase URL here as well?

Reply:
That's actually a good question. I added it to one application, when testing secondary IdP, but I didn't add it to the other application, when testing primary IdP, so maybe it's not required? I'll give it another test.

Edit: It worked without the Firebase URL, so I'm going to leave it out of the instructions.